mobile/lib/infrastructure/repositories/sync_api.repository.dart

```diff
@@ -1,27 +1,31 @@
 import 'dart:async';
 import 'dart:convert';
 
-import 'package:flutter/foundation.dart';
+import 'package:http/http.dart' as http;
 import 'package:immich_mobile/domain/interfaces/sync_api.interface.dart';
 import 'package:immich_mobile/domain/models/sync/sync_event.model.dart';
 import 'package:immich_mobile/services/api.service.dart';
+import 'package:logging/logging.dart';
 import 'package:openapi/api.dart';
-import 'package:http/http.dart' as http;
 
 class SyncApiRepository implements ISyncApiRepository {
+  final Logger _logger = Logger('SyncApiRepository');
   final ApiService _api;
-  const SyncApiRepository(this._api);
+  SyncApiRepository(this._api);
 
   @override
   Stream<List<SyncEvent>> watchUserSyncEvent() {
-    return _getSyncStream(
-      SyncStreamDto(types: [SyncRequestType.usersV1]),
-    );
+    return _getSyncStream(SyncStreamDto(types: [SyncRequestType.usersV1]));
+  }
+
+  @override
+  Stream<List<SyncEvent>> watchPartnerSyncEvent() {
```

Comment:
If this return type isn't even specific to users, or partners, or any other type, I don't see any benefit in having this interface be copied for each type. At this point we'd be better to just have `getSyncEvents(types: SyncRequestType[])` and letting the business service pass in the request type.